web/packages/teleport/src/Bots/Add/Shared/GuidedFlow.tsx

```diff
@@ -50,7 +50,7 @@ export function GuidedFlow({ name, title, views, icon }: FlowProps) {
 
   function handleNextStep() {
     if (currentStep < steps - 1) {
-      setCurrentStep((currentStep += 1));
+      setCurrentStep(currentStep += 1);
```

Comment:
i still think this should be just `currentStep + 1` 😅  (no assignment `equal`)

Reply:
Oh, I missed it in one place, it is fixed now, ty